examples/acados_template/python/test/test_data/generate_c_code_out_LS_FULL_CONDENSING_QPOASES_IRK_SQP.json

```diff
@@ -0,0 +1 @@
0.01568483617170517, -0.051357920596701495], [1.9781132352066064, -0.03436041451908119, 0.017687960915684694, -0.0406732778751358], [1.976708942924815, -0.03578676280231161, 0.01865244620241953, -0.030557423255412535], [1.975255499576003, -0.03682349422125712, 0.018744921916193955, -0.02127082810483241], [1.9737677726235765, -0.0375076656117764, 0.018131625243218076, -0.012992875870143635], [1.9722590231547092, -0.03788205959141867, 0.016972372219971176, -0.005831048309588519], [1.9707407318617738, -0.037992500762142035, 0.01541593996503428, 0.00016904714777526714], [1.9692225242220776, -0.03788558173172294, 0.013596751368104246, 0.005013329617094115], [1.9677121783476492, -0.03760680282089517, 0.011632729760919464, 0.008749012731754874], [1.966215699149101, -0.03719911354007269, 0.009624175344252063, 0.011454839849337933], [1.9647374431937372, -0.036701831495684664, 0.007653506958591306, 0.0132320552900533], [1.9632802798060212, -0.036149905188974825, 0.0057857113183319864, 0.014196302350192811], [1.9618457754335525, -0.03557348096705979, 0.004069346162422304, 0.014470567754567672], [1.9604343899713592, -0.03499773076463799, 0.0025379572706458508, 0.014179234960714947], [1.959045675498956, -0.034442896491707654, 0.001211751837569606, 0.013443222351188652], [1.9576784696425595, -0.03392450696650285, 9.94729296140643e-05, 0.01237623477729931], [1.956331077517781, -0.033453725510430535, -0.0007996981308573653, 0.01108198233358508], [1.9550014377887635, -0.0330377904121692, -0.0014942494684108706, 0.009652316908898844], [1.9536872698342245, -0.032680513790835454, -0.0019985329259206096, 0.008166167855160063], [1.9523862002943486, -0.032382809141486085, -0.0023310945516810534, 0.00668915992091164], [1.9510958683665505, -0.03214322259093298, -0.0025131761939292284, 0.005273794397331306], [1.9498140101219905, -0.03195844762548197, -0.0025674099538834246, 0.003960077027204925], [1.9485385228323837, -0.031823807603607764, -0.002516715893504073, 0.002776482573294659], [1.9472675108381348, -0.03173369460534134, -0.002383403889494858, 0.0017411550620065872], [1.9459993148686379, -0.03168195700277435, -0.0021884728516724187, 0.0008632537419053141], [1.9447325269612659, -0.03166223150186727, -0.0019510945999726284, 0.00014436695936249683], [1.9434659932365455, -0.03166821827250427, -0.0016882653918399975, -0.0004200712024621005], [1.9421988067934657, -0.0316939001435123, -0.001414605250220778, -0.0008394141005332769], [1.9409302929108962, -0.03173370870413089, -0.0011422836809151606, -0.0011265260430329718], [1.9396599885979866, -0.03178264155090353, -0.0008810498950633573, -0.0012966254474737773], [1.938387618346117, -0.03183633588864253, -0.0006383460743081701, -0.0013662668544979714], [1.9371130677136612, -0.03189110428354885, -0.00041948334380636647, -0.0013524721613559576], [1.935836356136666, -0.031943938628072995, -0.0002278617734475813, -0.001272013935019068], [1.9345576101155901, -0.03199248836491653, -6.521774721030303e-05, -0.0011408475172811708]]}
```

Comment:
did you check that all these combinations give reasonable trajectories?

Reply:
Just checking acados status. If you have an automatic way to check I can implement it ;)